pkg/sql/sqlbase/structured.proto

```diff
@@ -606,6 +606,24 @@ message TableDescriptor {
   // Mutation jobs queued for execution in a FIFO order. Remains synchronized
   // with the mutations list.
   repeated MutationJob mutationJobs = 27 [(gogoproto.nullable) = false];
+
+  message SequenceOpts {
+    // How much to increment the sequence by when nextval() is called.
+    optional int64 increment = 1 [(gogoproto.nullable) = false];
+    // Minimum value of the sequence.
+    optional int64 min_value = 2 [(gogoproto.nullable) = false];
+    // Maximum value of the sequence.
+    optional int64 max_value = 3 [(gogoproto.nullable) = false];
+    // Start value of the sequence.
+    optional int64 start = 4 [(gogoproto.nullable) = false];
+    // How many values to pre-allocate and cache in memory (not currently supported)
+    optional int64 cache = 5 [(gogoproto.nullable) = false];
```

Comment:
If we're not planning to support the cache option anytime soon we probably don't need to add this to the proto, especially not as a non-nullable field. An extra int64 per descriptor obviously won't kill us, though. And since descriptors are cached this might just not be worth worrying about.

Reply:
Agreed. Removed it.